katello-installer/katello-installer-base.spec

```diff
@@ -16,16 +16,22 @@ Source0: https://github.com/Katello/katello-installer/archive/%{version}%{?preve
 BuildArch: noarch
 Obsoletes: katello-installer < 2.1.0
 
-Requires: foreman-installer >= 1.11.0
+Requires: foreman-installer >= 1.12.0
 Requires: %{?scl_prefix}puppet >= 3.4.0
 Requires: katello-selinux
 Requires: openssl
 Requires: katello-certs-tools
 Requires: foreman-proxy
 
+BuildRequires: %{?scl_prefix}puppet >= 3.4.0
+BuildRequires: %{?scl_prefix}rubygem-kafo >= 0.8.0
+%if 0%{?fedora} >= 22
+BuildRequires: %{?scl_prefix}rubygem-puppet-strings
+%endif
```

Comment:
Since we don't build for Fedora, I suppose this requirement isn't needed in our case?